### PR TITLE
e2e: serial: create workload with TAS followed by default scheduler

### PR DESCRIPTION
**Pull request by swatisehgal**
Create a workload using TAS scheduler that is scheduled followed by
another similar workload using the default scheduler that stays pending.

Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
CI failures look legit
```
# github.com/openshift-kni/numaresources-operator/test/e2e/serial/tests
Error: test/e2e/serial/tests/workload_placement.go:845:28: baseload.CPU undefined (type baseload.Load has no field or method CPU)
Error: test/e2e/serial/tests/workload_placement.go:848:25: baseload.CPU undefined (type baseload.Load has no field or method CPU)
Error: test/e2e/serial/tests/workload_placement.go:851:28: baseload.Memory undefined (type baseload.Load has no field or method Memory)
Error: test/e2e/serial/tests/workload_placement.go:854:25: baseload.Memory undefined (type baseload.Load has no field or method Memory)
make: *** [Makefile:114: vet] Error 2
Error: Process completed with exit code 2.
```

**Comment by fromanirh** [comment]
I'd be in favour of adding `CPU()` and  `Memory` helpers to `baseload.Load` if this helps

File: internal/baseload/baseload.go [hunk 1]
```diff
@@ -103,3 +103,11 @@ func (nl Load) ToResourceList() corev1.ResourceList {
 	nl.Apply(res)
 	return res
 }
+
+func (nl Load) CPULoad() resource.Quantity {
```
Comment (fromanirh) [inline internal/baseload/baseload.go:107]: I'm still very much in favor of adding these, but I'm surprised github is not complaining - we merged https://github.com/openshift-kni/numaresources-operator/pull/361 which changed the internal representation, so this PR actually needs a rebase

Comment (swatisehgal) [reply internal/baseload/baseload.go:107]: Yep, on it.

**Comment by swatisehgal** [comment]
Updates:
1. Implemented this test case with a pod instead of a deployment.
2. Determined resources available per NUMA node and used them as a request for the two containers instead of the implementation eliminating the need for `leastAvailableResourceQtyInAllZone` and `FindResourceAllocatableByname`. That inturn meant that there was no need to expose baseload CPU and memory. These changes would be needed and implemented in https://github.com/openshift-kni/numaresources-operator/pull/327.
3. This PR was tested locally and is passing.

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**